apps/condo/domains/common/utils/sberCloudFileAdapter.js

```diff
@@ -0,0 +1,72 @@
+const ObsClient = require('esdk-obs-nodejs')
+class SberCloudFileAdapter {
+
+    constructor (config) {
+        this.bucket = config.bucket
+        this.s3 = new ObsClient(config.s3Options)
+        this.server = config.s3Options.server
+        this.folder = config.folder
+    }
+
+    save ({ stream, filename, id, mimetype, encoding }) {
+        return new Promise((resolve, reject) => {
+            const fileData = {
+                id,
+                originalFilename: filename,
+                filename: this.getFilename({ id, originalFilename: filename }),
```

Comment:
Cloud you please generate a random name

Reply:
It is already random as it includes Id in getFileName function